app/models/job_invocation.rb

```diff
@@ -204,17 +204,12 @@ def output(host)
   def generate_description
     key_re = /%\{([^\}]+)\}/
     template_invocation = pattern_template_invocations.first
-    hash_base = Hash.new { |hash, key| hash[key] = "%{#{key}}" }
-    input_hash = template_invocation.input_values.reduce(hash_base) do |h, v|
+    input_hash = template_invocation.input_values.reduce({}) do |h, v|
       h.update(v.template_input.name => v.value)
```

Comment:
If we set the keys here to the replacement format like:
```suggestion
      h.update("%{#{v.template_input.name}}" => v.value)
```
Then below we can do 
```ruby
self.description = description_format.gsub(/%{[^}]+}/) { |key| input_hash[key] }